Changelog.md

```diff
@@ -7,6 +7,7 @@ Compiler Features:
  * AST: Export NatSpec comments above each statement as their documentation.
  * Inline Assembly: Do not warn anymore about variables or functions being shadowed by EVM opcodes.
  * Optimizer: Simple inlining when jumping to small blocks that jump again after a few side-effect free opcodes.
+ * Yul EVM Code Transform: Stack Optimization: Reuse slots of unused function arguments and defer allocating stack slots for return variables until after any expression statements and assignments that do not reference them.
```

Comment:
"after all"?

Reply:
Didn't I change this from ``all`` to ``any`` earlier :-)? I'll just remove it, that should be fine in any case.

Reply:
maybe :)